Jellyfin.Plugin.TelegramNotifier/Jellyfin.Plugin.TelegramNotifier.csproj

```diff
@@ -12,7 +12,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.*-*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.9.7" />
+    <PackageReference Include="Jellyfin.Model" Version="10.9.7" />
```

Comment:
```Jellyfin.Model``` is not used within this plugin.

Reply:
This was added automatically when installing dotnet package dependencies.